sklearn/utils/estimator_checks.py

```diff
@@ -74,10 +74,10 @@
                 'OrthogonalMatchingPursuit', 'PLSCanonical', 'PLSRegression',
                 'RANSACRegressor', 'RadiusNeighborsRegressor',
                 'RandomForestRegressor', 'Ridge', 'RidgeCV']
-
 ALLOW_NAN = ['Imputer', 'SimpleImputer', 'MissingIndicator',
              'MaxAbsScaler', 'MinMaxScaler', 'RobustScaler', 'StandardScaler',
              'PowerTransformer', 'QuantileTransformer']
+SUPPORT_STRING = ['SimpleImputer', 'MissingIndicator']
```

Comment:
surely OneHotEncoder, OrdinalEncoder and meta-estimators will belong here too.

Reply:
Can we do that in another PR. I think that we should also factorize the input validation as well. It is quite redundant and we could have a common test then.

Reply:
Yes, it probably belongs in estimator tags... (perhaps post-#8022)